src/View/SSViewer_Scope.php

```diff
@@ -339,12 +345,26 @@ public function next()
     public function __call($name, $arguments)
     {
         $on = $this->itemIterator ? $this->itemIterator->current() : $this->item;
+        if (is_scalar($on)) {
```

Comment:
This seems like kind of the wrong place for this, ideally casting would happen in viewable data